src/components/Category/Category.jsx

```diff
@@ -0,0 +1,30 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import { categories } from "../../staticData";
+import "./Category.css";
+
+export const Category = () => {
+	return (
+		<>
+			<div className="text-base font-semibold flex-row content-space-between mx-5 gutter-bottom">
+				<div className="category-header font-semibold">Categories</div>
+				<Link to={"/products"} className="flex-row flex-center">
+					<span className="category-header-btn font-semibold">Shop now</span>{" "}
+					<span class="material-icons-outlined font-base ml-1">east</span>
+				</Link>
+			</div>
+			<article className="category">
+				{categories.map(({ alt, image, category }, index) => (

```

Comment:
There is `/api/categories` route. Don't we need to use it here and add category information in db?

Reply:
I will add in the next PR.